src/core/sentinel.wat

```diff
@@ -0,0 +1,28 @@
+(module
```

Comment:
It would be nice to add a module comment to let the reader know why this module is needed, and when we can remove it.